frontend/components/CustomBurgerIcon/CustomBurgerIcon.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Why is the file called CustomBurgerIcon.jsx instead of index.jsx?

Reply:
Changed that :-)